packages/legend-art/style/reset/_mui.scss

```diff
@@ -64,3 +64,24 @@
 .MuiTooltip-tooltipPlacementTop {
   margin: 0.5rem 0;
 }
+
+.mui-radio-group {
```

Comment:
Let's move this block up to the top of the file, this section belongs to overriding global MUI styles